common/lib/dependabot/credential.rb

```diff
@@ -0,0 +1,30 @@
+# typed: strict
```

Comment:
Can this be `strong`?

Reply:
strong gets mad that `def_delegators` brings in untyped methods. I decided to punt on figuring that out or reimplementing them like with `[]` for now. I think the PR got large enough 😬